packages/pixelboy/apps/journal/checkin/checkin.coffee

```diff
@@ -47,3 +49,34 @@ class PAA.PixelBoy.Apps.Journal.CheckIn extends AM.Component
 
       else
         FlowRouter.go 'journal'
+
+  getExternalImgUrl: (event) ->
+    url = event.currentTarget.value
+    externalImgUrl = null
+    imgPreview = $('.image-preview')
+    placeholder = 'http://placehold.it/350x150?text=your+art'
+    errorIndicator = $('.preview-error')
+
+    if /twitter\.com/.test(url)
+      id = url.split('/status/')[1]
+      Meteor.call 'getImgFromTweet', id, (error, data) =>
+        if !error
+          externalImgUrl = data.entities.media[0].media_url_https
+          imgPreview.attr('src', externalImgUrl)
```

Comment:
Just to bring up an exception with parentheses, here the parentheses are correctly included since this is happening on a jQuery object (which would be evident if the variable would be called `$imagePreview`, one more reason for the `$` prefix convetion). We choose to do parentheses with jQuery since function chaining is an often used practice and this way your code is prepared to chain if needed.